Assets/js/results.js

```diff
@@ -87,15 +100,15 @@ function makeListCard(arrInd){ //what is feeding into here is the object that is
         movieCard.setAttribute('class', 'movieCard');
         movieCard.setAttribute('id', arrInd); // added by henry
         movieCard.setAttribute('style', 'display:flex; align-items: center; border: 1px solid white; margin: 20px; padding: 10px')
-        // movieCard.setAttribute('onclick', 'goToInfo(event)')
-        // movieCard.onclick = (event) => {
-        //     let currentDI = event.target.getAttribute('data-index')
-        //     console.log(currentDI);
-        // };
 
         createH4.textContent = movieObj.title;
         let subText = 'released ' + releaseYear + ' | ' + genres + '';
         createH5.textContent = subText;
+        let newObj = {
```

Comment:
can we use a more descriptive name instead of newObj here?

Reply:
we can if you want, but if we change the way we get data for the info page, it will be unnecessary.